build/gen_check.sh

```diff
@@ -48,5 +48,6 @@ EOF
 
 # Run the tests
 test_gen 'go run ./cmd/policygen --config_path=examples/policygen/config.yaml --state_path examples/policygen/example.tfstate --output_path' 'examples/policygen/generated'
-test_gen 'go run ./cmd/tfengine --config_path=examples/tfengine/simple.hcl --output_path' 'examples/tfengine/generated/simple'
```

Comment:
After Martin's PR is in, can we send a PR first to just do the renaming from simple to org_foundation so we don't see a lot of deletion in this one.

Reply:
Good idea, sent #332